.github/actions/get_test_infos/get_test_infos.sh

```diff
@@ -4,33 +4,12 @@
 DIRECTORIES=($INPUT_DIRECTORIES)
 EXCLUDE_DIRS=($INPUT_EXCLUDE_DIRS)
 
-# Function to discover test directories
-discover_directories() {
-    local dir=$1
-    find "$dir" -type d -not -path "*/node_modules/*" -exec sh -c '
-        for pkg in "$@"; do
-            if [ -f "$pkg/package.json" ]; then
-                if jq -e ".scripts.test" "$pkg/package.json" > /dev/null; then
-                    echo "$pkg"
-                fi
-            fi
-        done
-    ' sh {} +
-}
+# Convert arrays to comma-separated strings for passing into Node.js
+directories=$(IFS=, ; echo "${DIRECTORIES[*]}")
```

Comment:
What is this?

Reply:
It's setting the variable IFS to the value of `,`. IFS stands for Internal Field Separator, by setting it to be a comma when we expand DIRECTORIES it will separate them all with a comma